code/controllers/subsystem/movement/movement_types.dm

```diff
@@ -111,7 +113,13 @@
 	var/visual_delay = controller.visual_delay
 	var/success = move()
 
-	SEND_SIGNAL(src, COMSIG_MOVELOOP_POSTPROCESS, success, delay * visual_delay)
+	var/we_do_a_little_lying = success
```

Comment:
Is this temporary? This shouldn't exist and should just set success directly

Reply:
It is. I need potatos feedback on this. Its a workaround on a bug in their system